src/apidoc/config/template.xhtml

```diff
@@ -1,159 +1,163 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml"
-      xmlns:ml="http://developer.marklogic.com/site/internal"
-      xml:lang="en" lang="en">
-  <head>
-    <title><ml:page-title/></title>
-    <meta charset="utf-8" />
-    <meta name="description"
-          content="MarkLogic is the only Enterprise NoSQL Database" />
-    <meta name="keywords"
-          content="marklogic, enterprise nosql database, enterprise nosql, database, nosql, nosql database, nosql db, xml, xml database, json, enterprise, bigdata, big data, xquery, xslt, petabyte, java db, java database, content, content store, content database, content db, content management system, CMS, document, document-oriented databases, document database, document db, document store, DB, xml database, xml db, json db, nonrelational, nonrelational database, nonrelational db" />
-    <meta name="viewport" content="initial-scale=1.0" />
-    <link href="/css/bootstrap.min.css" rel="stylesheet"/>
-    <link href="/css/v-2/apidoc/jquery.treeview.css"
-          rel="stylesheet" type="text/css" media="screen" />
-    <!-- START ORBIT CSS -->
-    <ml:common-nav-css/>
-    <!-- END ORBIT CSS -->
-    <link href="/css/v-38/apidoc-screen.css"
-          rel="stylesheet" type="text/css" media="screen" />
-    <link href="/css/v-33/apidoc.css"
-          rel="stylesheet" type="text/css" media="screen" />
-    <link href="/css/v-1/apidoc_print.css"
-          rel="stylesheet" type="text/css" media="print" />
+xmlns:ml="http://developer.marklogic.com/site/internal"
```

Comment:
Looks like you lost a lot of the indentation here in the header. Could you restore that, please?